build.gradle

```diff
@@ -20,14 +20,13 @@ plugins {
 
 allprojects {
     repositories {
+        // Needs to go first to get specialty libraries https://stackoverflow.com/a/48438866/137744
+        google()
+
         jcenter()
-        maven { url = 'https://oss.sonatype.org/content/groups/public' }
-        maven {
```

Comment:
This one goes away because it's the same as `google()`: https://stackoverflow.com/questions/46467561/difference-between-google-and-maven-url-https-maven-google-com